pkg/resources/cloudconfig/openstack/cloudconfig.go

```diff
@@ -116,6 +132,15 @@ func (c *CloudConfig) String() (string, error) {
 	bs := out.Section("BlockStorage", "")
 	c.BlockStorage.toINI(bs)
 
+	for name, opts := range c.LoadBalancerClass {
```

Comment:
Iteration over the map is randomized, the resulted ini could be different in different reconciliations, possibly triggering re-deployment of the CCM. Keys need to be sorted.

```suggestion
	for _, name := range slices.Sorted(maps.Keys(c.LoadBalancerClass)) {
	    opts := c.LoadBalancerClass[name]
```

Reply:
Thanks for pointing this! updated the code accordingly in ead18848a7d5ff25ef376d81355b9fd12cb956f2